package.json

```diff
@@ -2,22 +2,24 @@
   "name": "redux-persist",
   "version": "5.6.9",
   "description": "persist and rehydrate redux stores",
-  "main": "lib/index.js",
-  "module": "es/index.js",
+  "main": "dist/redux-persist",
+  "module": "dist/redux-persist.es",
   "types": "./src/index.d.ts",
   "repository": "rt2zz/redux-persist",
   "files": [
     "es",
     "src",
-    "lib"
+    "lib",
```

Comment:
lib and es will go away with this correct?

Reply:
actually not, i thought at first they would, but with code splitting on we do not have control over output file names so we need to output files targeting cjs and esm into 2 separate directories to avoid the file path clash